app/workers/feeds/import_articles_worker.rb

```diff
@@ -4,8 +4,19 @@ class ImportArticlesWorker
 
     sidekiq_options queue: :medium_priority, retry: 10
 
-    def perform(earlier_than, user_ids = [])
-      users = user_ids.present? ? User.where(id: user_ids) : nil
+    # NOTE: [@rhymes] we need to default earlier_than to `nil` because sidekiq-cron,
+    # by using YAML to define jobs arguments does not support datetimes evaluated
+    # at runtime
+    def perform(earlier_than = nil, user_ids = [])
```

Comment:
I thought about splitting this in two separate jobs (as it was in the past) but: I'm not sure it's worth it, and I didn't want to bother to do another migration (we need to keep the two parameters for a while, exhaust existing jobs and then remove the second parameter).

I'm open to change it or to better ideas

Reply:
dumb question, can you elaborate on what you mean by having two separate jobs here and what would we benefit from it?

Reply:
The second argument right now it's only used when we fetch the feed from the settings page (when the user saves the profile information or explicitly requests a re-fetch), adding a separate job would cleanup the `if`, but nothing more. That's why I'm not super convinced about adding it

Reply:
Thanks for explaining! yeah, I agree it's good as is.